### DISCUSSION TITLE
Prevent Vercel for GitLab integration comments

### DISCUSSION THREAD

**Discussion by DanielRudrich** [vercel/community#3120] [Help] [answered]
Hi there,

we are using Vercel with Gitlab projects and very happy with it so far!

However, it would be nice to turn off the automatic comments on merge requests made by the GitLab integration similar to: https://vercel.com/guides/how-to-prevent-vercel-github-comments

Naively, we've tried 
```json
{
    "gitlab": {
        "silent": true
    }
}
```

But this didn't do the trick ;-)

Our team would be very happy if this setting could be added (or maybe a more general one to turn off both at once?).

In case we simply did something and wrong and that setting alread exists, sorry for the noise :) 

Cheers
Daniel

**Comment by humza024** [comment]
how to enable comments ?

**Reply by swarnava** [reply]
Hi, Can you create a separate thread using https://github.com/orgs/vercel/discussions/new?category=help ? You can learn more about enabling or disabling comments [here](https://vercel.com/docs/workflow-collaboration/comments/specialized-usage#enable-or-disable-comments-for-branches)

**Comment by swarnava** [comment, marked as answer]
Hi,

Unfortunately, we don't have an option to silent GitLab comment at this moment. However, this is something we are working on for all Git Integration. We will announce it in vercel.com/changelog once this is enabled. :) Thank you!
